nix/overlay.nix

```
@@ -0,0 +1,47 @@
+final: prev: 
+(with builtins; seq ((hasAttr "inputs" prev) || throw "If you are calling this directly, make sure the overlays have an `inputs` attribute conforming to the flakes interface."))
+(let
+  packages = final.inputs.self.packages.${prev.system};
+in {
```

```suggestion
inputs: final: prev: 
{
```
There is never a reason to use `packages` in overlays. Please use `prev` argument instead

---

I'm...not even sure why that's there. That should almost definitely be final.